balboa-agent/src/main/java/com/socrata/balboa/agent/HttpOrMq.java

```diff
@@ -0,0 +1,19 @@
+package com.socrata.balboa.agent;
+
+public enum HttpOrMq {
```

Comment:
Why is this a Java class? Socrata is transitioning to Scala, so new code should be in Scala if at all possible.